### ISSUE TITLE
selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document

### ISSUE THREAD

**Issue by Stripling07** [mintapi/mintapi#266]

**Comment by Stripling07** [comment]
Sorry, thought I had added the details, here is the error I get after I input the mfa code:

  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/mintapi/api.py", line 515, in __init__
    self.login_and_get_token(email, password,
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/mintapi/api.py", line 601, in login_and_get_token
    self.driver, self.status_message = get_web_driver(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/mintapi/api.py", line 434, in get_web_driver
    _sign_in(email, password, driver, mfa_method, mfa_token, mfa_input_callback, intuit_account, wait_for_sync, wait_for_sync_timeout, imap_account,
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/mintapi/api.py", line 410, in _sign_in
    driver.find_element_by_id("ius-sign-in-mfa-password-collection-current-password").send_keys(password)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 477, in send_keys
    self._execute(Command.SEND_KEYS_TO_ELEMENT,
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document
  (Session info: chrome=91.0.4472.114)

**Comment by jaron-l** [comment]
Can you confirm what version of mintapi you are using?

**Comment by BlakeHolifield** [comment]
I also saw this: 
```
pip3 freeze | grep mintapi
mintapi==1.49
```

There are some stackoverflow threads about handling that exception, but throwing a `time.sleep(5)` after the initial `Mint()` object creation seemed to be a gnarly but effective workaround.

**Comment by jaron-l** [comment]
@BlakeHolifield do you have a semi-reliable way to reproduce? I can't get the error. It happens at the same location as mentioned in #265 which is in a try except loop trying to detect a login method, what I don't get is that they both are finding the element appropriately (so passing the test that the try is testing for) but then the element behaves diffently than expected. Either the finding of the element is not thorough enough or the page is dynamically changing when we don't expect it to.

**Comment by BlakeHolifield** [comment]
@jaron-l I just tried to reproduce and couldn't 🙃 . With or without the sleep, it seems fine now. I'm _thinking_ it might have to do with the `Syncing account status` message? When I first logged in that message took a solid 15 seconds to finish the accounts sync and that's when this first occurred. Since then that sync is much faster (2s or less) and I haven't been able to reproduce. Sorry I don't have more to go on here. If it helps, I'm using 

- Firefox 89
- OSX 10.15.7
- Python 3.8.5
